examples/platform/silabs/efr32/rs911x/rsi_if.cpp

```diff
@@ -724,7 +749,7 @@ void ProcessEvent(WfxEvent_t inEvent)
                 if (!wfx_rsi.scan_ssid || (wfx_rsi.scan_ssid && strcmp(wfx_rsi.scan_ssid, (char *) scan->ssid) == CMP_SUCCESS))
```

Comment:
I know you didn't touch this, but could you tweak this strcmp(wfx_rsi.scan_ssid, (char *) scan->ssid) to make sure it doesn't compare more than the smaller buffer in case those strings not null terminated.

Reply:
done